Kombit.Samples.CH.WebsiteDemo/MyPage.aspx.cs

```diff
@@ -117,7 +122,14 @@ protected static bool ValidateKombitAttributeProfile(Saml20Identity current)
             if (!current.HasAttribute("dk:gov:saml:attribute:CvrNumberIdentifier"))
```

Comment:
Need to use the new claim type

Reply:
updated with OIOSAML 3 required claim types